src/runtime/Makefile

```diff
@@ -159,7 +159,66 @@ DEFMEMSZ := 2048
 DEFMEMSLOTS := 10
 #Default number of bridges
 DEFBRIDGES := 1
-DEFENABLEANNOTATIONS := []
+DEFENABLEANNOTATIONS := [			\
```

Comment:
:skull_and_crossbones: :radioactive: ? This has to be the scariest / coolest PR title we've had in a while! :smile:

How about tweaking slightly to something like:

```make
# Comments explaining:
#   1. What constitutes "non-dangerous".
#   2. When this should be updated (do we need a comment in `annotations.go` btw?)
#   3. The impact of values not specified in this list.
NON_DANGEROUS_ANNOTATIONS := [ ... ]

DEFENABLEANNOTATIONS := $(NON_DANGEROUS_ANNOTATIONS)
```

Also, could you sort that list alphabetically?

I'm vaguely wondering whether we should auto-generate `virtcontainers/pkg/annotations/annotations.go` and this list from a YAML file maybe.

Reply:
Will do, I'll get back to this one Today and address all the comments.
Thanks for the reviews!